### ISSUE TITLE
qrep example fixes

### ISSUE THREAD

**Issue by jacobrosenthal** [woboq/qmetaobject-rs#62]
I have no idea how to submit to qrep cgit so Ill complain to you :)

Looks like spmc 0.2 branch has basically entirely been yanked. 
spmc = "0.3" seemed to work fine though

I also needed
```
let (sender, receiver) = spmc::channel();
```
to be 
```
let (mut sender, receiver) = spmc::channel();
```

**Comment by ogoffart** [comment]
I suppose this should be reported to @vandenoever

**Comment by ratijas** [comment]
Has been fixed in 0b8eb36655f1b5918ebb0cf30e488892c523e365 via #125 (and probably broken many times since, 'cause you know, CI doen't test those patches).
